packages/analyticsdx-template-lint/src/schemas/template-info-schema.json

```
@@ -679,6 +679,30 @@
                 "body": "${0}"
               }
             ]
+          },
+          "liveConnection": {
+            "type": "object",
+            "additionalProperties": false,
+            "required": ["connectionName", "sourceObjectName"],
+            "properties": {
+              "connectionName": {
+                "type": "string",
+                "description": "The devname of the live connection (choose something unique to avoid collisions)"
+              },
+              "sourceObjectName": {
+                "type": "string",
+                "description": "The source object name"
+              }
+            },
+            "defaultSnippets": [
+              {
+                "label": "Live Dataset",
```

The snippet label shows in the code-completion item, and I've been using this convention:
```suggestion
                "label": "New liveConnection",
```